src/displayapp/screens/settings/Settings.h

```diff
@@ -41,6 +41,7 @@ namespace Pinetime {
           {Symbols::clock, "Date&Time", Apps::SettingSetDateTime},
           {Symbols::batteryHalf, "Battery", Apps::BatteryInfo},
           {Symbols::clock, "Chimes", Apps::SettingChimes},
+          {Symbols::clock, "Quiet Hour", Apps::SettingQuietHour},
```

Comment:
You need to bump the `nScreens` variable to 4.

Reply:
thank you. fixed.